python/sdk/prelude_sdk/models/codes.py

```diff
@@ -483,6 +499,7 @@ class PolicyType(Enum, metaclass=MissingItem):
     DEVICE_COMPLIANCE = 11
     IDENTITY_MFA = 12
     HOST_FIREWALL = 13
+    NETWORK_FIREWALL = 16
```

Comment:
idk if you're leaving room for something, but this PR is probably next to go in, so I'd jsut squish up the numbers